src/main/java/frc/team4909/robot/subsystems/climber/commands/SetStiltPosition.java

```diff
@@ -36,8 +36,9 @@ protected void execute() {
         double driveStiltNegSpeed = Robot.driverGamepad.getThresholdAxis(BionicF310.LT)  //Stilt drive backwards
         * RobotConstants.climberDriveSpeedManual;
 
-        Robot.climberSubsystem.setStiltsDriveSpeed(RobotConstants.climberDriveSpeedAuto); // Stilts via climnber
-        
+        if(Robot.elevatorSubsystem.getPosition() > 0){
+            Robot.climberSubsystem.setStiltsDriveSpeed(RobotConstants.climberDriveSpeedAuto); // Stilts via climnber
```

Comment:
the positive is also now up, so that may change the, just FYI

Reply:
Good point. I'll change later